secrets.defaults.properties

```diff
@@ -0,0 +1,30 @@
+
+# Default values for the Mifos API
+# You can override these values by creating secrets.properties file in the root directory of the project
+
+PROTOCOL_HTTPS = https://
+
+API_ENDPOINT = demo.mifos.community
+
+API_PATH = /fineract-provider/api/v1/
+
+PORT = 80
+
+TENANT = default
+
+# Provide GEO API Key
+GEO_API_KEY = AIzaSyAZTZqvDGyyw21z2Ee7N-dE_WuZQwKL0bs
+
+DEMO_SERVER_CONFIG = "https://,demo.mifos.community,/fineract-provider/api/v1/,80,default"
```

Comment:
This is right way, we are going to keep this only, remove rest of it

Reply:
can you also add a comment to DEMO_SERVER_CONFIG

#Update DEMO_SERVER_CONFIG according below mentioned your server config 
#PROTOCOL_HTTPS = https:// 
#API_ENDPOINT = demo.mifos.community
#API_PATH = /fineract-provider/api/v1/
#PORT = 80
#TENANT = default